source/_integrations/http.markdown

```diff
@@ -105,6 +105,21 @@ http:
 
 The [Set up encryption using Let's Encrypt](/blog/2015/12/13/setup-encryption-using-lets-encrypt/) blog post gives you details about the encryption of your traffic using free certificates from [Let's Encrypt](https://letsencrypt.org/).
 
+<div class='note'>
+
+  When using a reverse proxy, you will need to enable the `use_x_forwarded_for` and `trusted_proxies` options. Requests from reverse proxies will be blocked after 2021.7 if these options are not set.
+  
+  ```yaml
+  http:
+    use_x_forwarded_for: true
+    trusted_proxies:
+      - 172.30.33.0/24
+      - 127.0.0.1
+      - ::1
```

Comment:
Considering users copy & paste in general, I think this is kinda risky.
Accepting whole IP blocks and the local loopback interfaces seems way too wide IMHO.

Do you really have a cluster of reverse proxies? Do you expect others to have that?

Reply:
For my setup, the nginx addon is actually at 172.30.33.4, but I included the whole /24 block for easy copy-paste. How could we specify the correct IP? Not sure that all users who would use a reverse proxy (especially an addon) will know the correct IP to state.

Removing the `::1` and `127.0.0.1` seems like a good idea.

Reply:
Not setting the correct IP, can become a security risk. For example, when using trusted network authentication, another entity on the network could send in a fake X-Forwarded-For header to get authenticated as any user.

Reply:
Removing `::1` and `127.0.0.1` and leaving the `172.30.33.0/24` matches the example directly above. I also added a comment that clarifies that the proxy server's IP should be added to the list.